documentation-site/pages/blog/drag-and-drop-list/index.mdx

```diff
@@ -0,0 +1,297 @@
+import metadata from './metadata.json';
+import Layout from '../../../components/layout';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+import {TwitterTweetEmbed} from 'react-twitter-embed';
+import Removable from '../../../examples/dnd-list/removable';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+<BlogImage
+  src="https://res.cloudinary.com/flycatcher/image/upload/v1572289775/listgif_igl9j7.gif"
+  alt="Base Web DnD List component"
+  caption="Base Web DnD List component"
+/>
+
+The drag and drop interaction was invented in the 1970's by Jef Raskin as a part of the Macintosh project. That's 40 years ago! So how come that making things draggable on the web is still so painful? We were recently looking for answers since our component library needs to support a sortable list and slider.
```

Comment:
```suggestion
The drag and drop interaction was invented in the 1970's by [Jef Raskin](https://en.wikipedia.org/wiki/Jef_Raskin) as a part of the Macintosh project. That's 40 years ago! So how come that making things draggable on the web is still so painful? We were recently looking for answers since our component library needs to support a sortable list and slider.
```